x-pack/legacy/plugins/uptime/server/lib/adapters/saved_objects/kibana_saved_objects_adapter.ts

```diff
@@ -4,22 +4,48 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { APICaller } from 'kibana/server';
 import { UMSavedObjectsAdapter } from './types';
-import uptimeIndexPattern from './heartbeat_index_pattern.json';
+import {
+  IndexPatternsFetcher,
+  IIndexPattern,
+} from '../../../../../../../../src/plugins/data/server';
 
 export const savedObjectsAdapter: UMSavedObjectsAdapter = {
-  getUptimeIndexPattern: async client => {
+  getUptimeIndexPattern: async callES => {
```

Comment:
Since this is no longer a saved object does it still belong in the saved objects adapter?